composer.json

```diff
@@ -1,16 +1,25 @@
 {
     "name":        "hostnet/asset-bundle",
     "type":        "symfony-bundle",
-    "description": "Bundle to glue Assets",
+    "description": "Hostnet asset bundle allows for configuring the hostnet/asset-lib from within your symfony app",
     "license":     "MIT",
+    "authors": [
+        {
+            "name":  "Yannick de Lange",
+            "email": "ydelange@hostnet.nl"
+        }
+    ],
     "require": {
-        "php":               ">=7.1",
-        "hostnet/asset-lib": "dev-master",
-        "symfony/symfony":   "^3.3"
+        "php":                      "^7.1",
+        "hostnet/asset-lib":        "dev-master",
+        "symfony/framework-bundle": "^3.3",
+        "symfony/http-foundation":  "^2.7|^3.0",
```

Comment:
Does it really work with 2.7+? I suggest to put the minimal dependency on 3.3 like any other, will make upgrading to 3.4 and 4.0 a lot easier.